action/protocol/staking/vote_reviser.go

```diff
@@ -37,7 +39,32 @@ func (vr *VoteReviser) Revise(csm CandidateStateManager, height uint64) error {
 		}
 		vr.storeToCache(height, cands)
 	}
-	return vr.flush(height, csm)
+	return nil
+}
+
+func (vr *VoteReviser) CorrectAliasCands(csm CandidateStateManager, height uint64, owners CandidateList) error {
+	cands, ok := vr.cache[height]
+	if !ok {
+		csr := newCandidateStateReader(csm.SM())
+		var err error
+		cands, _, err = csr.getAllCandidates()
+		switch {
+		case errors.Cause(err) == state.ErrStateNotExist:
+		case err != nil:
+			return err
+		}
```

Comment:
these won't be used or even they are called, they cannot change anything.

Reply:
fixed